news.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About Page</title>
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" href="./news-style.css">
+    <link rel= "stylesheet" href = "nav.css">
+  </head>
+    <body>
+        <header>
+        
+            <nav>
+            <div class = "name">Ryan's Hair salon</div>
+            <ul class = "">
+                <li><a href = "./index.html">Home</a></li>
+                <li><a href = "./about.html">About</a></li>
+                <li><a href = "./gallery.html">Gallery</a></li>
+                <li><a href = "./news.html">News</a></li>
+                <li><a href = "./contact.html">Contact</a></li>
+            </ul>
+            </nav>
+          </header>
+
+          <div class = "container">
```

Comment:
no spaces between class, equals sign, and quotes. all attributes should like like this `<div class="container">`